static-src/scripts/maps/components/proposal-list-item.vue

```diff
@@ -145,7 +145,7 @@ export default ProposalListItem
     }
 
     input[type="range"] {
-        width: 5em;
+        width: 10em;
```

Comment:
This looks great, but it looks like now the opacity bars can overlap the text:
<img width="547" alt="Screen Shot 2019-07-22 at 2 13 19 PM" src="https://user-images.githubusercontent.com/13007381/61665867-f78a2180-ac8a-11e9-9f07-c4035c63249d.png">

If you inspect the `.collapsible-list-item-parent-left` element in the `collapsible-list-item` component, you'll see that it actually has a right-side padding of `5em`: this gives whatever is put into `collapsible-list-item-parent-right` `5em` of space.  So, if we're increasing the size of that to `10em`, we should also increase that right-side padding to `10em` and it'll prevent that overlap.  The line you'll need to change is [here](https://github.com/cestastanford/imaginedsf/blob/master/static-src/scripts/maps/components/collapsible-list-item.vue#L53).

This will unfortunately make the Proposal Map panel even larger as you've noticed, but we can work on that as part of the map redesign.